RELEASING.md

```diff
@@ -1,6 +1,5 @@
-1. Create a new virtual environment following the instructions in `CONTRIBUTING.md`. 
-   Make sure to also install all dependencies for the documentation including `altair_saver`
-   and uninstall `vl-convert-python` (this is not needed for normal contributions to the repo, see `CONTRIBUTING.md` for details).
+1. Create a new virtual environment following the instructions in `CONTRIBUTING.md`.
+   Make sure to also install all dependencies for the documentation.
```

Comment:
This is old right? We don't need altair_saver for the release anymore?

Also @jonmmease When I run `hatch test`, I run into an issue because vl-convert does not yet support the Vega-Lite version string. Does that mean we should wait for a vl-convert release before updating altair, so the order of releases needs to be `Vega-Lite` -> `vl-convert` -> `altair`?

Reply:
Yes, exactly. I'll watch for the Vega-Lite release and then publish vl-convert right after

Reply:
Then you'll want to update the vl-convert-python version string at:

https://github.com/altair-viz/altair/blob/5a7814cd14a2b336bea6e5b2bb7391467afe88fc/altair/utils/_importers.py#L32

It will probably be 0.14.0, but if Vega get's released as well I may bump it up to 1.0.0

Reply:
It is indeed 0.14.0

Reply:
Also, reminder to make this update in `_importers.py`.

Reply:
Thank you, updated!